python/src/aiconfig/eval/promptfoo/examples/travel/travel_parametrized.aiconfig.json

```diff
@@ -0,0 +1 @@
+../../../custom_eval/examples/travel/travel_parametrized.aiconfig.json
```

Comment:
what does this do having it here?

Reply:
It's just for convenience if your'e in the folder. I can clean up the folder structure and make paths behave nicer, at some point.